packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -112,6 +104,18 @@ const BuySellForm = props => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [input_amount, effective_rate]);
 
+    React.useEffect(() => {
+        if (exchange_rate !== previous_rate && rate_type === ad_type.FLOAT) {
+            setPreviousRate(exchange_rate);
+            showModal({
+                key: 'RateChangeModal',
+                props: {
+                    currency: buy_sell_store.local_currency,
+                },
+            });
+        }
+    }, [exchange_rate, previous_rate]);
```

Comment:
won't this give an eslint warning because of missing dependencies?

Reply:
Nope :grin: